src/jit/gentree.cpp

```diff
@@ -6586,115 +6799,165 @@ GenTreePtr Compiler::gtNewAssignNode(GenTreePtr dst, GenTreePtr src)
 }
 
 // Creates a new Obj node.
-GenTreeObj* Compiler::gtNewObjNode(CORINFO_CLASS_HANDLE structHnd, GenTree* addr)
+GenTree* Compiler::gtNewObjNode(CORINFO_CLASS_HANDLE structHnd, GenTree* addr)
 {
     var_types nodeType = impNormStructType(structHnd);
     assert(varTypeIsStruct(nodeType));
-    GenTreeObj* objNode = new (this, GT_OBJ) GenTreeObj(nodeType, addr, structHnd);
+    unsigned size = info.compCompHnd->getClassSize(structHnd);
```

Comment:
Note that impNormStructType() will call getClassSize() and getClassGCLayout() JIT-EE interface methods.
Afterwards, this routine is going to call getClassSize() and later there will be a call to getClassGCLayout() in  gtSetObjGcInfo().

Are we making the same JIT-EE call multiple times on a given structHnd while trying to create a blkop node?